loculus_values/environment_specific_values/production.yaml

```diff
@@ -32,7 +32,7 @@ secrets:
       username: AgCVsForafhGd6476UHYnWFs9TjZeikvgWteZIwdVRxRItdjBRyel3bPV/wJyKh/eIE7P6OtRVqdcM05ksDGVO5ptJrgiTT3v3QSQwf6y6fGrYK085qODRnHvU1HGq3Z58tQJRbr+Nv+pfKlHr+XemJ6SdGA4AFEsja+z+HLiPJzlYqwf63dbIT5WxxKBE4/Tucjr0ps0NvNPQqw3K/kXNf5IGKAr2OLh7GFQ1/StNpkWmQBUU31FvVMwQALkT1x32+2wdsfAGbazXsLxTlP9G600EpP3dVSaLwDTcS13eD9PyLwNd+ZRU4hOMWcVY2NSbswY7uSd4rHNeeuyXiBbAmTLwnVQrlfJ1Wq5TaCDqkGQR3zfl6EfQQNZKXhjKqfclFNdgwoYViDHzpj4JVFpKj5BXta14UCVuIKaKdyg6D6cb9UaOAMYrDqY8eHztS8oI41RklXZuk79NdU08dAnUu1pGir6SpBOZpwJqqKYIrr7OfTgTI38LkljLSTcErwb3YJx51cC2UdpVLIMWndCurn297djMtrMmcBNEOrS3s2pPMAshitO+ZfCRQwPQ7Y3pNAkS+yqkRRjOH2JIhxziya2waxnj3bL/fRNVo5NNbSKgFZXDZSceyQNKzwR3DguZDPNN3mZiOVmGUsxWJ7OPXtsMtcLR1M20eT16KHLUGceL5LvSs2Zs4yjFeNZJVrKAMN6XrEQpvZL6mW8mCFOw/VLw==
-      host: AgCk8CURGjV2raeTiwDAxn39zp1Ni0QM/1iHzBWyJBKzTaHl/GES6UZ+lUS+ZlWhi+XTgTGYuhHz4UJPnUNX3H47r6xczAdCsSRt37eZm3bOOObKGh+GS9E5Ju9Kr18q2wsGq2eJ6z9UCiPWS3okMwc6ciVFVymzRtFfqiDtdN0WBvnYI6raupV8Lw8mwoy4KATZXZRDejFKSKWEMsMjCDMEP6M8AoTzc6MfVKsg6DCazWm8LDzHsguJXlF5LjTEMsysZ6I3qT0WYmUVZcRjZxcJBzB1sXbkYyu9hZZYhG0kO/g/RfZDhcOTTEbkGHGiGAqqYt2SqVJzUMMVvHzYZOeU0eU/zRP9/MxlvTCiZJyG3CruZPyPobXWG71npXS2YwGmA7oL7g1GXuUAbOBwV1nYLL2djW7nXnchiZLkBYNmU8hDPHKRnWui94Cxz065RlvIRBoxq6hU0a5YJ6lhaldjIxqIBIeeqpwOUtiPGxcBPA9cmmj6vEidz8+DMXIHLJGjPHvRMKjohd5xMp+l0A/dX/Zym0nI8iBUvoQlMm8CNg4hpkJj+v6lBTJSkBYl6WKycEyGIugmMWZ+AbBSEw22qAfnmJNZM1UorWp7xYbGgN3ttdUK1ax/1qr8yeUFNXUWUB0OEmwIh8sgxpAWutIlHBfnN3rCpO+v2oDhYg64uEMGRWJeDDJoJaI67O32YGSzqCEkc7M0JcWZkEt0ax4DNWcammcXrHvON4pOmjCiHtO26eSpIPY/xHQlyoB9LOY2+z0c5r9mkLYoOqkOc/6X  
+      host: AgCk8CURGjV2raeTiwDAxn39zp1Ni0QM/1iHzBWyJBKzTaHl/GES6UZ+lUS+ZlWhi+XTgTGYuhHz4UJPnUNX3H47r6xczAdCsSRt37eZm3bOOObKGh+GS9E5Ju9Kr18q2wsGq2eJ6z9UCiPWS3okMwc6ciVFVymzRtFfqiDtdN0WBvnYI6raupV8Lw8mwoy4KATZXZRDejFKSKWEMsMjCDMEP6M8AoTzc6MfVKsg6DCazWm8LDzHsguJXlF5LjTEMsysZ6I3qT0WYmUVZcRjZxcJBzB1sXbkYyu9hZZYhG0kO/g/RfZDhcOTTEbkGHGiGAqqYt2SqVJzUMMVvHzYZOeU0eU/zRP9/MxlvTCiZJyG3CruZPyPobXWG71npXS2YwGmA7oL7g1GXuUAbOBwV1nYLL2djW7nXnchiZLkBYNmU8hDPHKRnWui94Cxz065RlvIRBoxq6hU0a5YJ6lhaldjIxqIBIeeqpwOUtiPGxcBPA9cmmj6vEidz8+DMXIHLJGjPHvRMKjohd5xMp+l0A/dX/Zym0nI8iBUvoQlMm8CNg4hpkJj+v6lBTJSkBYl6WKycEyGIugmMWZ+AbBSEw22qAfnmJNZM1UorWp7xYbGgN3ttdUK1ax/1qr8yeUFNXUWUB0OEmwIh8sgxpAWutIlHBfnN3rCpO+v2oDhYg64uEMGRWJeDDJoJaI67O32YGSzqCEkc7M0JcWZkEt0ax4DNWcammcXrHvON4pOmjCiHtO26eSpIPY/xHQlyoB9LOY2+z0c5r9mkLYoOqkOc/6X
```

Comment:
what changed here? line endings? can't see in GitHub mobile diff view

Reply:
yeah - old version had a space at the end of the line